plugins/commands/plugins/src/main/kotlin/PluginsCommand.kt

```diff
@@ -62,22 +62,17 @@ class PluginsCommand(descriptor: PluginDescriptor = PluginsCommandFactory.descri
         echo()
 
         plugins.forEach { plugin ->
-            echo(
-                HorizontalRule(
-                    buildString {
-                        append(plugin.displayName)
-                        if (plugin.id != plugin.displayName) append(" (id: ${plugin.id})")
-                    },
-                    "-"
-                )
-            )
+            echo(HorizontalRule(plugin.displayName))
             echo()
-            echo(plugin.description)
+
+            echo("ID: ${plugin.id}")
             echo()
 
-            if (plugin.options.isNotEmpty()) {
-                echo("Configuration options:")
+            echo("Description: ${plugin.description}")
```

Comment:
To me it seems redundant to add that prefix.

Reply:
As all other properties have prefixes / "key names" now, I wanted to be consistent here, and have an almost YAML-like output.